src/pages/Settings.tsx

```diff
@@ -297,6 +337,51 @@ export const Settings = () => {
     </>
   );
 
+  const exportKeysAsQrCodePage = (
+    <>
+      <BackButton onClick={() => setPage("main")} />
+      <Show
+        when={shouldVisibleExportedKeys}
+        whenFalseContent={<Text theme={theme}>Timed out. Please try again</Text>}
+      >
+        <ExportKeysAsQrCodeContainer>
+            <QrCode address={exportKeysQrData}/>
+        </ExportKeysAsQrCodeContainer>
+
+      </Show>
+    </>
+  );
+
+
+  const exportKeyOptionsPage = (
+    <Show
+      when={!showSpeedBump}
+      whenFalseContent={
+        <SpeedBump
+          theme={theme}
+          message={speedBumpMessage}
+          onCancel={handleCancel}
+          onConfirm={(password?: string) => handleSpeedBumpConfirm(password)}
+          showSpeedBump={showSpeedBump}
+          withPassword={decisionType === "export-keys" || decisionType === "export-keys-qr-code"}
+        />
+      }
+    >
+      <BackButton onClick={() => setPage("main")} />
+      <SettingsRow
```

Comment:
Put a wrapper around this or use some other means to bring the items towards the top rather than the center of the screen.